From b22fdef79292171c4aacdf6988bd8e8414426f89 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Tue, 3 Mar 2020 13:48:12 +0700
Subject: [PATCH 042/458] arm64: defconfig: enable Panel LVDS for RZ/G2 SoC

This patch enables config for supporting LVDS Panel in RZ/G2 SoC.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/configs/defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index e9f5c2f..4d2df0c 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -763,3 +763,6 @@ CONFIG_DRM_I2C_NXP_TDA998X=y
 # Enable Audio support
 CONFIG_SND_SOC_ALL_CODECS=y
 CONFIG_COMPILE_TEST=y
+
+# Enable LVDS support
+CONFIG_DRM_PANEL_LVDS=y
-- 
2.7.4

